validation/chOpenApiValidator.go

```diff
@@ -236,31 +238,29 @@ func findRoute(r routers.Router, req *http.Request) (route *routers.Route, pathP
 	return r.FindRoute(req)
 }
 
-func getSchema(ctx context.Context, openApiSpec, contextId string, chv *CHValidatorImpl) (err error) {
-	//callOnce.Do method ensures the code is executed once so that the schema is not repeatedly loaded unnecessarily.
-	callOnce.Do(func() {
-		log.InfoC(contextId, "Retrieving openAPI3 schema")
-		chv.doc, err = loadSchemaFromFile(ctx, openApiSpec, contextId)
-		if err != nil {
-			log.ErrorC(contextId, err, log.Data{config.OpenApiSpecKey: openApiSpec, config.MessageKey: "unable to open Open API spec"})
-		} else {
-			if err = chv.doc.Validate(ctx); err != nil {
-				log.ErrorC(contextId, err, log.Data{config.MessageKey: "error occurred while trying to call kin-openAPI validation method"})
-			}
+func getSchema(ctx context.Context, openApiSpec string) (*openapi3.T, error) {
+
+	log.Info("Retrieving openAPI3 schema")
```

Comment:
Picky, but the log messages use "schema" or "spec". Pick one. Probably spec, as that is CH lingo.

Reply:
Changed to spec 👍